src/weaviate/docker-compose.yaml

```diff
@@ -3,7 +3,7 @@ services:
   weaviate:
     image: semitechnologies/weaviate:1.20.5
     ports:
-      - 8080:8080
+      - 8081:8080
```

Comment:
Is there a reason for this change?

Reply:
Port 8080 is occupied when using jupyter on vertex workbench